vendor/engines/secure_rooms/spec/factories/secure_room.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :secure_room, class: SecureRoom do
```

Comment:
ah okay I added in the `parent:` but I think it also needs the `class:` to generate the right model